crypto/bio/bss_mem.c

```diff
@@ -168,7 +168,7 @@ static int mem_buf_sync(BIO *b)
     if((b != NULL) && (b->init) && (b->ptr != NULL)) {
```

Comment:
you missed one :)  and if you touch it, please remove the extra internal parens.